cirq-core/cirq/ops/gateset.py

```diff
@@ -501,12 +511,11 @@ def _validate_operation(self, op: raw_types.Operation) -> bool:
 
         The containment checks are handled as follows:
 
-        a) For any operation which has an underlying gate (i.e. `op.gate` is not None):
+        - For any operation which has an underlying gate (i.e. `op.gate` is not None):
             - Containment is checked via `self.__contains__` which further checks for containment
                 in any of the underlying gate families.
-
-        b) For all other types of operations (eg: `cirq.CircuitOperation`,
-        `cirq.GlobalPhaseOperation` etc):
+        - For all other types of operations (eg: `cirq.CircuitOperation`,
+            `cirq.GlobalPhaseOperation` etc):
```

Comment:
nit: `cirq.GlobalPhaseOperation` is now deprecated, so let's remove it from the docstring. 
```suggestion
            etc):
```